include/ada/url.h

```diff
@@ -228,6 +228,11 @@ namespace ada {
      */
     ada_really_inline bool parse_prepared_path(const std::string_view input);
 
+    /**
+     * Parse scheme assuming it ends with ':'
```

Comment:
This new condition (assuming it ends with ':') is a bit odd.

Reply:
It was wrong. I updated it.